src/server/game/AuctionHouse/AuctionHouseMgr.h

```diff
@@ -76,11 +76,8 @@ struct AuctionEntry
     uint32 bidder;
     uint32 deposit;                                         //deposit can be calculated only when creating auction
     AuctionHouseEntry const* auctionHouseEntry;             // in AuctionHouse.dbc
-    uint32 factionTemplateId;
 
     // helpers
-    uint32 GetHouseId() const { return auctionHouseEntry->houseId; }
```

Comment:
Wouldn't it be easier to keep this method and change it to `return houseId;`? As a away to avoid breaking the "API"